### PR TITLE
Keras added missing test cases in backend_test

### PR DESCRIPTION
**Pull request by PariksheetPinjari909**
Added test cases for following 

1. cast_to_floatx
2. identity
3. cumsum
4. cumprod
5. any
6. all

### CODE REVIEW

**Comment by gunan** [comment]
Please redirect the review to high level APIs/Keras owners.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by xiejw** [comment]
Thanks for the contributions. Looks reasonable to me. 
I would like fchollet to review.

**Review by karmel — DISMISSED** [review]
Thanks--

**Comment by gbaned** [comment]
@PariksheetPinjari909 can you please address Ubuntu Sanity build failures

**Comment by PariksheetPinjari909** [comment]
@gbaned Solved the failure
@karmel Can you please approve again. Thanks

**Comment by rthadur** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
